### PR TITLE
Index mobile

### PR DESCRIPTION
**Pull request by dwang733**
Make the letter navbar not show on mobile (phone or tablet). I also consolidated the tests for the index pages into one file, instead of three separate files, for clarity.

### CODE REVIEW

File: features/index_page.feature [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Feature: View the index pages
```
Comment (jlabowitz) [inline features/index_page.feature:1]: Good consolidation of the test files.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+	def mobile_device
```
Comment (jlabowitz) [inline app/helpers/application_helper.rb:2]: This is a good way to implement a mobile check.

**Review by jlabowitz — APPROVED** [review]

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (nrleuenb) [reply app/helpers/application_helper.rb:2]: Agreed, this is a good idea. Might be hard to apply to the overall formatting though.

**Review by nrleuenb — APPROVED** [review]
LGTM; Tests look good

**Pull request merged**